content/en/docs/Reference/catalog-templates.md

```diff
@@ -503,3 +503,340 @@ schema: olm.channel
 
 Here, a channel is generated for each template channel which differs by minor version, and each channel has a `replaces` edge from the predecessor channel to the next-lesser minor bundle version. Please note that at no time do we transgress across major-version boundaries with the channels, to be consistent with [the semver convention](https://semver.org/) for major versions, where the purpose is to make incompatible API changes.
 
+## Composite Template
+A `composite template` is an input schema that accepts a configuration schema for generating a valid FBC for multiple catalogs at a time. This approach may be attractive to catalog maintainers looking to build multiple versions of a catalog while maintaining fewer files and operator authors looking to contribute their operator to multiple catalogs.
```

Comment:
```suggestion
You can use a `composite template` to generate a valid FBC for multiple catalogs at a time. A `composite template` is an input schema that accepts a configuration schema. Catalog maintainers can use a `composite template` to build multiple versions of a catalog while maintaining fewer files. Operator authors can use a `composite template` to contribute their operator to multiple catalogs.
```

Reply:
I like the sound of this much better, thanks!